tests/python/test_continous_dynamics.py

```diff
@@ -89,6 +89,140 @@ def test_multibody_free():
         pass
 
 
+def test_centroidal():
+    try:
+        space = manifolds.VectorSpace(9)
+        nk = 2
+        nu = 3 * nk
+        mass = 10.5
+        gravity = np.array([0, 0, -9.81])
+        contact_map = [(True, np.array([0, 0.1, 0])), (True, np.array([0.1, -0.1, 0]))]
+        ode = dynamics.CentroidalFwdDynamics(space, nk, mass, gravity, contact_map)
+        data = ode.createData()
+
+        assert isinstance(data, dynamics.CentroidalFwdData)
+
+        x0 = space.neutral()
+        u0 = np.random.randn(nu)
+
+        ode.forward(x0, u0, data)
```

Comment:
It could be nice to test forward result.

Try to do a simple test with the CoM moving in X direction, Y direction, Z direction. To avoid only copying the C++ code in Python.